src/viam/robot/client.py

```diff
@@ -364,6 +400,15 @@ def resource_names(self) -> List[ResourceName]:
         with self._lock:
             return [r for r in self._resource_names]
 
+    def _close_channel(self, *, tab_count=0):
```

Comment:
(q) I'm not sure I get why the `tab_count` exists. Why do we want to indent when calling `close()` specifically?

Reply:
Because I call this internal function from a 2 difference places, and I want the logs to reflect a certain level of indentation depending on where it was called from (e.g. if it was called as part of a task that closes a lot of things, it should be indented. if it was closed on its own, it should not be indented)